### DISCUSSION TITLE
Update to draft 2020-12

### DISCUSSION THREAD

**Discussion by JamieKnow** [danielaparker/jsoncons#351] [General]
We're just about to start using JSONCONS. Do you have any plans to update with support for newer drafts?

**Comment by danielaparker** [comment]
We'd like to update JSON Schema support to a later draft, but there's no time line.

**Comment by JamieKnow** [comment]
Thanks Daniel,
Would you consider doing the work as a paid engagement?


Jamie Knowles
Senior Product Manager, ER/Studio
E: ***@***.******@***.***>
O: +1(713)341-2807 M: +44 7561-516517

[signature_1535105136]

www.ideracorp.com<https://www.ideracorp.com/>

IDERA<https://www.idera.com/>  •  AquaFold<https://www.aquafold.com/>  •  Webyog<https://www.webyog.com/>  •  WhereScape<https://www.wherescape.com/>  •  Qubole<https://www.qubole.com/>


From: Daniel Parker ***@***.***>
Reply to: danielaparker/jsoncons ***@***.***>
Date: Thursday, 17 February 2022 at 00:18
To: danielaparker/jsoncons ***@***.***>
Cc: JamieKnow ***@***.***>, Author ***@***.***>
Subject: Re: [danielaparker/jsoncons] Update to draft 2020-12 (Discussion #351)


We'd like to update JSON Schema support to a later draft, but there's no time lime.

—
Reply to this email directly, view it on GitHub<https://github.com/danielaparker/jsoncons/discussions/351#discussioncomment-2192630>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXYH4NZECJROGXXIJ4UDIK3U3Q5DTANCNFSM5OK3E3DQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by danielaparker** [comment]
Jamie,

Could be an option, you can contact me at danielaparker@yahoo.com.

Best regards,
Daniel
